docs/spectrum1d.rst

```diff
@@ -81,24 +85,37 @@ List of Loaders
 ~~~~~~~~~~~~~~~
 
 The `~specutils.Spectrum1D` class has built-in support for various input and output formats.
-A full list of the supported formats is shown in the table below. Note that the JWST readers
-require the ``stdatamodels`` package to be installed, which is an optional dependency for
-``specutils``.
+A full list of the supported formats is shown in the table below and
+can be accessed interactively with ``Spectrum1D.read.list_formats()``.
+Note that the JWST readers require the ``stdatamodels`` package to be
+installed, which is an optional dependency for ``specutils``.
 
 .. automodule:: specutils.io._list_of_loaders
 
-| More information on creating custom loaders can be found in the :doc:`custom loading </custom_loading>` page.
+Call the help function for a specific loader to access further documentation
+on that format and optional parameters accepted by the ``read`` function,
+e.g. as ``Spectrum1D.read.help('tabular-fits')``.
+
+More information on creating custom loaders for formats not covered
+by the above list can be found in the :doc:`custom loading </custom_loading>` page.
 
 Writing to a File
 -----------------
 
-Similarly, a `~specutils.Spectrum1D` object can be saved to any of the supported formats using the
-:meth:`specutils.Spectrum1D.write` method.
+Similarly, a `~specutils.Spectrum1D` object can be saved to any of the
+formats supporting writing (currently only the two generic FITS formats)
+by using the :meth:`specutils.Spectrum1D.write` method.
 
 .. code-block:: python
 
     >>> spec1d.write("/path/to/output.fits")  # doctest: +SKIP
 
+Note that the above example, calling ``write()`` without specifying
+any format, will default to the ``wcs1d-fits`` loader if the `~specutils.Spectrum1D`
+has a compatible WCS, and to ``tabular-fits`` otherwise, or if writing
+to another than the primary HDU (``hdu=0``) has been selected.
+For better control of the file type, the ``format`` parameter should be explicitly passed.
```

Comment:
does header need to be specified? or does it default to using what's in meta?

Reply:
Per default it is using `meta['header']` both in `tabular-fits` and `wcs1d-fits` (in principle it depends on the loader, but those 2 are the only ones that provide a writer at the moment).
With `update_header=True` it treats the entire `meta` as a dictionary to update the default header from (i.e. pulls in all items that are serialisable and do not conflict with FITS keyword rules).